napari/_qt/containers/_base_item_model.py

```diff
@@ -29,7 +29,7 @@
 )
 
 
-class _BaseEventedItemModel(QAbstractItemModel, Generic[ItemType]):
+class _BaseEventedItemModel(Generic[ItemType], QAbstractItemModel):
```

Comment:
It would be great to have this one line merged even if the rest is still a WIP. 

This one seems good on its own given the current state of Qt.